tools/run_tests/run_interop_tests.py

```diff
@@ -1318,7 +1329,7 @@ def aggregate_http2_results(stdout):
                                 service_account_key_file,
                                 transport_security='tls')
                             jobs.append(tls_test_job)
-                            if language == 'c++':
+                            if str(language) in ['c++', 'go']:
```

Comment:
sorry, one more thing:

we need to pass: `auth=True` to this `cloud_to_prod_jobspec` invocation, similarly as is done above for c++

Without passing `auth=True`, this script won't pass the args like --default_service_account, needed for these OAuth tests

Reply:
done